pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -822,38 +824,40 @@ func handleCreate(ctx *domainContext, key string, config *types.DomainConfig) {
 	// Do we need to copy any rw files? !Preserve ones are copied upon
 	// activation.
 	for _, ds := range status.DiskStatusList {
-		if ds.ReadOnly || !ds.Preserve {
-			continue
-		}
-		log.Infof("Copy from %s to %s\n", ds.FileLocation, ds.ActiveFileLocation)
-		if _, err := os.Stat(ds.ActiveFileLocation); err == nil {
-			if ds.Preserve {
-				log.Infof("Preserve and target exists - skip copy\n")
-			} else {
-				log.Infof("Not preserve and target exists - assume rebooted and preserve\n")
-			}
-		} else {
-			if err := cp(ds.ActiveFileLocation, ds.FileLocation); err != nil {
-				log.Errorf("Copy failed from %s to %s: %s\n",
-					ds.FileLocation, ds.ActiveFileLocation, err)
-				status.PendingAdd = false
-				status.LastErr = fmt.Sprintf("%v", err)
-				status.LastErrTime = time.Now()
-				publishDomainStatus(ctx, &status)
-				return
+		if !status.IsContainer {
```

Comment:
I suspect the diffs are just about identation, but I refuse to verify that using my eyes. From what I can tell the change could have been done by inserting three lines (and leaving identation unchnged):
 if status.IsContainer {
     continue
}

Please confirm that is the case.

Reply:
Yes, for container case, this portion of the code is being ignored.
Would like to revisit to check whether something has to be done in container case.

Is "leaving indentation unchanged" allowed within a if statement?

Reply:
Yeah -- that's what @nordmark said -- just insert that  if before the code block and leave the indentation be @spatchineelam 

This is the kind of cosmetics that makes it much easier to review and I think we should take care of it before merge.

Reply:
Taken care of